transport/whisper/filter.go

```diff
@@ -58,25 +58,25 @@ type Chat struct {
 	Listen bool `json:"listen"`
 }
 
-func (c *Chat) IsPublic() bool {
+func (c *Filter) IsPublic() bool {
 	return !c.OneToOne
 }
 
-type ChatsManager struct {
+type filtersManager struct {
 	whisper     *whisper.Whisper
 	persistence *sqlitePersistence
 	privateKey  *ecdsa.PrivateKey
-	keys        map[string][]byte // a cache of symmetric keys derived from passwords
+	keys        map[string][]byte // a cache of symmetric manager derived from passwords
```

Comment:
@adambabik: what is a "symmetric manager"?

Reply:
Good catch. My editor must have changed that when using "Refactor" feature. I will change this back to `keys`.